config/settings.yml

```diff
@@ -4,7 +4,7 @@ binaries:
   pdftk: pdftk
   clamdscan: /usr/bin/clamdscan
 
-hostname: www.example.com
+hostname: localhost:3000
```

Comment:
I need to make this change to get the proper URL, but is hostname properly set in all other environments? Why was this ever made www.example.com in the first place?

Reply:
@jkassemi @wyattwalter any reason for this to not be localhost:3000 and is it properly set for other environments on dev, staging, and prod? I would think so since pagination links would otherwise be broken.

Reply:
Ha, is that Rails boilerplate maybe? I see no issue with it being localhost:3000 by default.

Reply:
Good find. Only seeing this used for the swagger docs, and that should be functioning for local now with this change. 👍